.github/workflows/zz_generated.gitleaks.yaml

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT. Generated with:
```

Comment:
@stone-z Should we remove this file?

Reply:
I see no harm in having it in `releases` -- WDYT?

Reply:
Fine with me. We should eventually get it from our `github` dev automation though I guess.